exist-core/src/main/java/org/exist/xquery/functions/fn/FunSerialize.java

```diff
@@ -128,12 +128,13 @@ private static boolean isSerializationParametersElement(final Item item) {
 
     /**
      * Sequence normalization as described in
-     * http://www.w3.org/TR/xslt-xquery-serialization-30/#serdm
+     * <a href="http://www.w3.org/TR/xslt-xquery-serialization-30/#serdm">...</a>
```

Comment:
I found another occurrence of `...`

Reply:
Thanks @line-o I just pushed a commit which fixes that.